dll/win32/msv1_0/msv1_0.c

```diff
@@ -1281,6 +1290,10 @@ LsaApLogonUserEx2(IN PLSA_CLIENT_REQUEST ClientRequest,
             LogonInfo->Password.MaximumLength = 0;
         }
 
+        // decode password
```

Comment:
I would suggest to follow the existing code style regarding the comments (except maybe when they are TODO comments).

Reply:
done